x/ccv/consumer/types/params.go

```diff
@@ -52,41 +51,35 @@ func DefaultParams() Params {
 
 // Validate all ccv-consumer module parameters
 func (p Params) Validate() error {
+	if err := ccvtypes.ValidateBool(p.Enabled); err != nil {
+		return err
+	}
+	if err := ccvtypes.ValidatePositiveInt64(p.BlocksPerDistributionTransmission); err != nil {
+		return err
+	}
+	if err := ccvtypes.ValidateString(p.DistributionTransmissionChannel); err != nil {
+		return err
+	}
+	if err := ccvtypes.ValidateString(p.ProviderFeePoolAddrStr); err != nil {
```

Comment:
Isn't there a better way to validate an address?

Reply:
Good call, see `validateProviderFeePoolAddrStr`